### PR TITLE
python-ccrawl

### PR DESCRIPTION
**Pull request by dualfade**
https://github.com/BlackArch/blackarch/pull/2760

please review.

### CODE REVIEW

File: packages/python-mmh3/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-mmh3
```
Comment (noraj) [inline packages/python-mmh3/PKGBUILD:4]: Yet another git branching issue

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-ccrawl/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-ccrawl
+pkgname=('python-ccrawl')
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:5]: don't use parenthesis when there is only one element in the array

File: packages/python-ccrawl/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-ccrawl
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:4]: pkgbase used only when you do 2 in 1 package like py2 + py3 lib, so if only py3 no pkgbase

File: packages/python-ccrawl/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+pkgname=('python-ccrawl')
+_pkgname=ccrawl
+pkgver=1.2
+pkgrel=1
+pkgdesc='Ccrawl uses clang to build a database related to various C/C++ data structures.'
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:9]: Don't put the pkgname in the desc.

File: packages/python-ccrawl/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:11]: not groups for dependencies

File: packages/python-ccrawl/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+url='https://github.com/bdcht/ccrawl'
+license=('GPL3')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/bdcht/$_pkgname.git")
+sha512sums=('SKIP')
+
+prepare() {
+  cp -a "$_pkgname"{,-2}
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:19]: remove `{,-2}` since you don't package the py2 version

Comment (noraj) [reply packages/python-ccrawl/PKGBUILD:19]: Also remove double quote when there is no no dashes or slashes

File: packages/python-ccrawl/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+build() {
+  cd "$srcdir/$_pkgname"
+
+  python setup.py build
+}
+
+package_python-ccrawl() {
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:28]: put only `package`, `package_python-ccrawl` is when you have 2 in 1 package

File: packages/python-ccrawl/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  depends=('python' 'python-virtualenv' 'python-click' 'python-traitlets' \
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:29]: It seems that `python-virtualenv` is not required but python-pymongo or python-requests are missing https://github.com/bdcht/ccrawl/blob/release/requirements.txt

Also maybe python-python-rapidjson and python-pyparsing https://github.com/bdcht/ccrawl/blob/bbb434bce370c2e42044225a6a91ba7b15e8d58e/setup.py#L28-L35

File: packages/python-ccrawl/PKGBUILD [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+    'python-tinydb' 'clang')
+
+  cd "$_pkgname"
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:32]: no double quotes here

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-ccrawl/PKGBUILD [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+pkgname=('python-ccrawl')
+_pkgname=ccrawl
+pkgver=1.2
+pkgrel=1
+pkgdesc='Ccrawl uses clang to build a database related to various C/C++ data structures.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='https://github.com/bdcht/ccrawl'
+license=('GPL3')
+makedepends=('git' 'python-pip')
```
Comment (noraj) [inline packages/python-ccrawl/PKGBUILD:14]:
```diff
-python-pip
+python-setuptools
```

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-ccrawl/PKGBUILD (hunk 8 above)
Comment (dualfade) [reply packages/python-ccrawl/PKGBUILD]: python-python-rapidjson pukes. just fyi.

File: packages/python-ccrawl/PKGBUILD (hunk 6 above)
Comment (dualfade) [reply packages/python-ccrawl/PKGBUILD]: This should be removed with no py2 deps ? If so; will resolve. Thanks.

**Pull request closed without merging**